InternalTestApp/PrebidMobileDemoRendering/ViewControllers/Adapters/Prebid/AdMob/PrebidAdMobBannerViewController.swift

```diff
@@ -246,4 +247,20 @@ class PrebidAdMobBannerViewController:
         stopRefreshButton.isEnabled = false
         adUnit?.stopRefresh()
     }
+    
+    private func setProccesArgumentParser() {
+        let processArgumentsParser = ProcessArgumentsParser()
+        processArgumentsParser.addOption("ADD_USER_DATA", paramsCount: 2) { [weak self] params in
```

Comment:
Were these originally added to support seller-defined audiences? If so, it may mean these are deprecated because they add to the `user.ext` object. However, if there is other data that gets added to ext that doesn't belong in `user.data` then this stays.

Reply:
@CJxD, nope, this code takes the app launch params and puts them into the SDK properties. It is used in the integration tests. We start the app with some params and check whether these params are present in the bid request. In this particular case, we take the "old" params, but we use the new method `addUserData` to add them.